store/generated_key/memstore/README.md

```diff
@@ -48,6 +49,18 @@ $ curl http://localhost:3100/memstore/config | \
   curl -X PATCH http://localhost:3100/memstore/config -d @-
 ```
 
+### Instructed Status Code Return
+The instructed status code return allows users to set a desired returned status code for some payloads the user is about to write. The next time when a user requests to get the payloads with keys, the proxy returns the result corresponding to the status code set earlier. The status code is sticky, and
```

Comment:
wdym by "user is about to write"? Isn't there no write? You just set this option and then reads return this error.